python/tests/unit/arcticdb/version_store/test_filtering.py

```diff
@@ -1010,6 +1010,25 @@ def test_filter_numeric_isnotin_hashing_overflow(lmdb_version_store):
     assert_frame_equal(df, result)
 
 
+_uint64_max = np.iinfo(np.uint64).max
+
+
+@pytest.mark.parametrize("op", ("in", "not in"))
+@pytest.mark.parametrize("signed_type", (np.int8, np.int16, np.int32, np.int64))
+@pytest.mark.parametrize("uint64_in", ("df", "vals") if PANDAS_VERSION >= Version("1.2") else ("vals",))
```

Comment:
Why the limitation with the Pandas version?

Reply:
See the similar change on line 971. Pandas older than 1.2.0 (e.g. 1.1.5, the last version for Python 3.6) exhibit the same buggy behaviour when signed and unsigned mix.